po/eo.po

```diff
@@ -0,0 +1,236 @@
+# Esperanto translation for Ladybird.
+msgid ""
+msgstr ""
+"Project-Id-Version: ladybird wip\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-08-27 00:38+0300\n"
+"Language: eo\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 "
+"&& n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
```

Comment:
Pretty sure that's not how plurals in Esperanto work :)

It should be 
```suggestion
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Reply:
Whoops! Got a little too excited with Ctrl-C.